datahandling.py

```diff
@@ -13,71 +13,58 @@
 import regex
 
 
-# methods to load files and filter data in them:
+def _load_pickle(path, encoding='utf-8'):
+    with open(path, mode="rb") as f:
+        try:
+            return pickle.load(f, encoding=encoding)
+        except UnicodeDecodeError:
+            f.close()
+            os.remove(path)
+
+            if "apicalls" in path.lower():
+                return {}
+
+            if "latestmessages" in path.lower():
+                return {GlobalVars.meta_tavern_room_id: [], GlobalVars.charcoal_room_id: [],
+                        GlobalVars.socvr_room_id: [], '111347': []}
+
+            if "bodyfetcher" in path.lower():
+                return {}
+        except Exception as e:
+            raise e
```

Comment:
Catching an exception just so you can `raise` it is useless.  Just take out the `except Exception` branch.